packages/at_lookup/lib/src/at_lookup_impl.dart

```diff
@@ -423,7 +423,9 @@ class AtLookupImpl implements AtLookUp {
   }
 
   @override
-  Future<bool> pkamAuthenticate() async {
+  Future<bool> pkamAuthenticate(
```

Comment:
@gkc 
Found this issue while addressing review comment today.
if a caller using pkam with secure element creates an instance of at_lookup and calls
at_lookup.executeCommand or at_lookup.executeVerb without calling at_lookup.pkamAuthenticate(signingAlgo, hashingAlgo), auth will fail since executeCommand or executeVerb use default auth mechanism. 
Can I move signingAlgo and hashingAlgo from pkamAuthenticate to constructor ?

Reply:
Yeah makes sense. As optional parameters?

Reply:
yes

Reply:
There is clientConfig map already part of AtLookup constructor. So I made use of this instead of optional params

Reply:
in onboarding cli, at_lookup is created within at_client--> remote_secondary.So passing the signing and hashing algo won't work. Trying another approach by adding a setter to at_lookup